docs/react-v9/contributing/dev-workflow.md

```diff
@@ -1,4 +1,100 @@
+
+That said, some team members use [SourceTree](https://www.sourcetreeapp.com/), [GitHub app](https://desktop.github.com/) or the git integrations available in VS Code.
+
+We don't have an official branch naming policy. However some developers use the following to make their branches easier to find: `user/[alias]/your-fancy-branch-name`, ie `user/johndoe/fix-that-bug`.
+
+### Building
+
+The inner development loop usually involves these steps:
+
+First, `cd` to the root folder of the repo (usually `fluentui`) and run `yarn` to install dependencies and link packages. This can take some time.
+
+```
+cd fluentui
+yarn
+```
+
+Run `yarn start` and select your start up project.
+
+```
+@fluentui/public-docsite // starts the public v8 documentation site.
+@fluentui/public-docsite-v9 // starts the internal v9 documentation storybook site.
+```
+
+Pick a specific component package if you intend to work on a v9 component directly.
+
+Most projects have hot module reloading, so file saves will trigger refreshes.
+
+### Iterating
+
+Make the changes you need, and commit along the way.
+
+```
+git status // Shows all changed files
+git add . // Stages the changed files
+git commit -m "Your brief message." // Makes the commits. You may notice a slight lag during committing as our linters work away.
+git push upstream // Pushes your changes to the central Fluent UI repo.
+```
+
+### Syncing with master
+
+It is strongly recommended that you rebase your branch onto (rather than merging with) master.
+
+```
+git checkout master // Switches to master
+git pull upstream master // Syncs your local master with the latest version of master at the origin
+git checkout your-fancy-branch // Switches to your branch
+git rebase master // Tacks your commits onto the end of master.
+```
+
+Resolve any conflicts in your editor.
+
+`git push upstream` // Pushes your changes to the central Fluent UI repo.
+
+Creating _draft_ pull requests is often an easy way to keep track of your work without it being actively reviewed by others.
+
+In other cases, such as before checking in or running tests, you may need to run a full build (or build up to a certain package):
+
+- `yarn build` - build everything. It is good the run this weekly, or anytime you start a new project off master.
+- `yarn buildto package-name` - build up to a package.
+  - `yarn buildto @fluentui/react`
+
+You can also `cd` to any package under `packages/*` and run `yarn build` to build individual things, though keep in mind that this may require dependencies to be built first (using `build --to`).
+
+### Making a pull request
+
+Make sure all your changes are committed, and run a build on the package that was updated, e.g. `yarn workspace @fluentui/react-components build` for changes to the v9 components. This will compare your changes and detect if the publicly facing API has changed, and update the right docs accordingly. Commit this change.
```

Comment:
One last thought -- it might make sense to mention that if a PR includes any changes to the styles or output HTML, they'll need to update snapshots by running either `yarn update-snapshots` or `yarn just jest ComponentName -u`

Reply:
just a suggestion though. If you think that's better off in a testing file, that works too!

Reply:
added :-)